modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc

```diff
@@ -0,0 +1,24 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/getting-started-with-openshift-gitops.adoc
+
+[id="logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials_{context}"]
+= Logging in to the Argo CD instance by using your OpenShift credentials
+
+Red Hat OpenShift GitOps Operator automatically creates a ready-to-use Argo CD instance that is available in the `openshift-gitops` namespace.
+
+.Prerequisites
+
+* You have installed the Red Hat OpenShift GitOps Operator in your cluster.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, navigate to _Operators_ -> _Installed Operators_ to verify that the Red Hat OpenShift GitOps Operator is installed.
+. Navigate to _Red Hat applications_ -> _Application Stages_ -> _Argo CD_. The login page of the Argo CD UI is displayed in a new window.
```

Comment:
Click the (app launcher icon.png) icon and navigate...
_Red Hat applications_ -> _Application Stages_ -> _Argo CD_
All GUI elements need to be bold. GH uses markdown where *xyz* renders into italics, and **xyz** renders into bold. Adoc renders *xyz* into bold. We need GUI elements in bold not italics.